webpack.config.js

```diff
@@ -32,9 +32,7 @@ module.exports = (env, argv) => {
 
     const entry = overrides.entry || {main: './src/index.js'};
 
-    const devtool = overrides.devtool || (
-        mode === 'development' ? "eval-source-map" : 'none'
-    );
+    const devtool = overrides.devtool || 'source-map';
```

Comment:
default to source-map now -- this causes the dev bundle to increase dramatically in number of lines, even if it is smaller now -- same issue as with https://github.com/plotly/dash-renderer/pull/105#issuecomment-445050463